types/artifacts.d.ts

```diff
@@ -161,7 +161,7 @@ declare global {
         lhId?: string,
         devtoolsNodePath: string,
         selector: string,
-        boundingRect: Rect | null,
+        boundingRect?: Rect,
```

Comment:
This is the functional change to the PR. In practice it shouldn't be observable because I believe `null` could only ever come from `link-elements` and `script-elements` and we wouldn't have been trying to put their boundingRects in the LHR in the first place, but you never know where some code is checking against `undefined` instead of falsiness or something.